features/steps/utils.py

```diff
@@ -139,6 +141,30 @@ def _clean(loggable):
     return data
 
 
+def traverse(resource, path):
+    def walk(data, k):
+        if isinstance(data, dict):
+            return data.get(k)
+        elif isinstance(data, list):
+            return [reduce(walk, [k], el) for el in data]
+        return None
+    return reduce(walk, path, resource)
+
+
+def is_field_in_resource(resource, fields):
```

Comment:
Naming nit: singular field in name vs are at argument. Maybe call the `some_field_present` or `any_field_present` or `has_one_of`?